### ISSUE TITLE
It's bad to not have just-an HTML file, or to print to a PDF... and NOTHING is there...

### ISSUE THREAD

**Issue by metadings** [arvida/emoji-cheat-sheet.com#337]
How do you save `emoji-cheat-sheet.com`?

**Comment by buli1433** [comment]
![images 3](https://cloud.githubusercontent.com/assets/9019289/11774174/5eadb0f8-a22a-11e5-9bd1-64255f82cb86.jpg)

**Comment by arvida** [comment]
@metadings True. We should do a print stylesheet. If you got the time, please look in to it :wink:

**Comment by krokofant** [comment]
I'm able to print it to PDF by enabling background graphics.
![image](https://cloud.githubusercontent.com/assets/5908498/16595360/0939c402-42ef-11e6-9115-29583d81bd47.png)
